src/functions/context/api.tsx

```diff
@@ -29,16 +31,43 @@ export const deleteAPI = async (id: string) => {
 
 export const getAllAPI = async (orgID: string) => {
   if (isFlagEnabled('managed-functions')) {
-    await getApiV2privateFunctions({query: {orgID}})
-    // if (res.status != 200) {
-    //   throw new Error(res.data.message)
-    // } TODO
-    return {
-      functions: [
-        {name: 'functionb', id: '1', orgID: '0', script: 'lalal'},
-        {name: 'functiona', id: '2', orgID: '0', script: 'lalal'},
-      ],
+    const res = await getApiV2Functions({query: {orgID}})
+    if (res.status != 200) {
```

Comment:
lol we seperated `notebooks` and `notebooks-api` flags so that we could independantly test the ux and the api layer as we were developing them. if notebooks-api was off, it would generate ids on saves and noop on updates and removals, while updates and adds still worked on local storage. when the flag was on, we'd still write to local storage, but also send off requests to the backend

Reply:
i do like this rollover mechanism in lieu of that though, just for dev and demo purposes